src/Utils/getFundToDisplay.js

```diff
@@ -0,0 +1,22 @@
+import { groupBy } from 'lodash';
```

Comment:
for common code we use `src/common/` dir, `Utils` is obsolete

Reply:
Moved to common folder